### PR TITLE
[WNMGDS-1736] updated doc site footer

### PR DESCRIPTION
**Pull request by scheul93**
## Summary

Simplifying the footer on the doc site to match new designs.

## How to test

`yarn start:gatsby` to see the changes in action
Screen Shot 2022-07-18 at 10 19 57 AM

### CODE REVIEW

File: packages/docs/src/styles/components/footer.scss [hunk 1]
```diff
@@ -1,88 +1,35 @@
 .c-footer {
-  .ds-footer__container {
-    margin-inline: auto;
+  border-top: 2px solid $color-gray-lighter;
+  padding: 1.5rem 1.5rem 0;
```
Comment (line47) [inline packages/docs/src/styles/components/footer.scss:3]: Can we match the spacing of the article element of `ds-u-sm-padding-x--6` for the footer?

**Review by forrestbaer — APPROVED** [review]
Looks good! 👍🏻

**Comment by zarahzachz** [comment]
I spotted some funkiness with the underline in the button link (mgov only). It's not offset the same as the neighboring anchor links and doesn't seem to have a hover effect to denote interactivity.

Screen Shot 2022-07-22 at 10 18 47 AM

**Review by zarahzachz — COMMENTED** [review]
I love seeing deleted code! Just the one observation on the button link (I don't know if it's part of this work though)

**Comment by scheul93** [comment]
> I spotted some funkiness with the underline in the button link (mgov only). It's not offset the same as the neighboring anchor links and doesn't seem to have a hover effect to denote interactivity.

@zarahzachz , updated! It looks like the link underline thickness and offset is configurable per theme

Screen Shot 2022-07-22 at 10 58 30 AM

**Pull request merged**